app/src/ble.c

```diff
@@ -141,6 +146,7 @@ bool zmk_ble_active_profile_is_connected() {
         return err;                                                                                \
     }
 
+/* Start direct advertising to peer if no connection exists for active profile */
```

Comment:
I believe direct advertising to hosts is currently not a thing because apple do not permit it to any apple device